src/dcrypto/dcrypto.go

```diff
@@ -55,6 +55,9 @@ var hashers map[Version]hasher
 // a hash of a local file.
 var MaxHeaderSize = v1.HeaderSize + 4
 
+// Overhead is the overhead added by the preferred encryption library plus the version.
+var Overhead = v1.Overhead + 4
```

Comment:
In that case we always know that the version itself contributes 4 bytes right? Could we later on make this say `const SizeOfVersion = 4`?